server/lists/listController.js

```diff
@@ -0,0 +1,43 @@
+var List = require('./ListController');
+var mongoose = require('mongoose');
+
+module.exports = {
+  getAllLists: function (id, next) {
+    List.find({tripId: id})
+      .then(function (trips) {next(trips);
+      })
+      .catch(function (err) {
+        next(err);
```

Comment:
Errors should be handled inside this function. Currently, "next" will be passed either the trips or the error as the first argument. This will cause problems with our callback because it won't know whether the argument being passed in is a  trip or an error.